kazoo/tests/test_threading_handler.py

```diff
@@ -200,6 +200,28 @@ def wait_for_val():
         eq_(lst, [True])
         th.join()
 
+    def test_wait_race(self):
+        mock_handler = mock.Mock()
+        async_result = self._makeOne(mock_handler)
+        async_result.set("immediate")
+
+        cv = threading.Event()
+
+        def wait_for_val():
+            try:
+                # NB: should not sleep
+                async_result.wait(10)
+                cv.set()
+            except ImportError:
+                pass
```

Comment:
why is this needed?

Reply:
Copied from test_wait(). I'll remove it.